pkg/kubelet/eviction/helpers.go

```diff
@@ -145,6 +145,10 @@ func parseThresholdStatements(statements map[string]string) ([]evictionapi.Thres
 	if len(statements) == 0 {
 		return nil, nil
 	}
+	// explicit none means no thresholds
+	if _, ok := statements[string(evictionapi.SignalNone)]; ok {
+		return nil, nil
```

Comment:
I think we want an empty list rather than nil here.  We use the resulting list like this: `results = append(results, hardThresholds...)`, where `hardThresholds` is the list of thresholds returned by this function.

Reply:
Why should the return value for 'none' be any different than len(statements) == 0?
Appending nil has the same effect as appending an empty slice.

Reply:
Oh, neat.  TIL